ROADMAP OCT18-MAR19.md

```diff
@@ -1,3 +1,20 @@
-## Data Analytics
+## New User Experience -davidm
+* Investigate from a UX/design perspective how we want to handle our own signup vs. Slack signup
+ * Is there a way we can combine the two? -davidm
+ * Explore removing autoinvite from Heroku and bringing into our current infrastructure -conrad
+ * Have an indicator on our landing page that we are on Slack -davidm
+ * Verification of new member re. Slack for security -nell
+
+## Infrastructure -ashley
+* Assess other infrastructure options besides Kubernetes
+ * Now? (the platform) -kyle
+
+## Data Analytics -conrad
 * Retrieve user demographic data from Facebook and LinkedIn social media login
```

Comment:
@jjhampton - we just have facebook and google
@dmarchante - we will need a backend lead to shepherd this